### ISSUE TITLE
Slideshow

### ISSUE THREAD

**Issue by tobiasKaminsky** [owncloud/android#1235]
My recents use cases are to show photos with a usb-> hdmi adapter.
I use the resized images, currently found in beta, so there is no option to use another image app.

My suggestion is
- add slideshow button when image is shown as option
- show dialog with time intervall (in seconds), random
- loop is always enabled

What do you think?

**Comment by AndyScherzinger** [comment]
I am a month late, but it sounds like a neat feature and a fitting addition to the image view itself even if it is just for showing a slideshow of your vacation pictures to friends on your phone. So yeah, sounds great! :+1:

**Comment by bxabi** [comment]
Yes, this would allow me to use a tablet as photo frame for owncloud, no need to copy all pictures to the tablet.

**Comment by tobiasKaminsky** [comment]
@jancborchardt what do you think?

**Comment by bxabi** [comment]
in the meanwhile I found that the QuickPic Gallery app and some others already support owncloud as image and slideshow source, and that way I can have multiple sources in the same app. 
So the importance of the feature is lower.

**Comment by jancborchardt** [comment]
I think it’s a very specific usecase as a photo frame. Normally when showing vacation pics to friends you advance manually.

Also, we should first focus on other basics (like performance improvements) before adding this. Otherwise the app has a strange imbalance. ;)

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are
closing it as outdated. If this is still relevant on a current ownCloud release,
please **reopen** (or open a fresh issue) with up-to-date details — thank you\!
